### ISSUE TITLE
ROM/Gamelist Problem in most recent test build

### ISSUE THREAD

**Issue by neosquid** [dankan1890/mewui#7]
Not sure if you are aware of this (or looking for bug reports on the test build), but the new test build doesn't seem to process roms very well, only recognizing about 25% of my roms as listed in the available games list. Once the roms in the rom folder are deleted, the games remain in the available list. This doesn't happen in the release build.

**Comment by dankan1890** [comment]
It seems that in committing this morning I forgot to update the "r" version.
It also happens with the current test build?

**Comment by neosquid** [comment]
Downloaded the latest test build. It still only detects 200 out of 1000 roms, whereas the release version detects 1000 roms.

I noticed that the test build now seems to clear the list of roms in the mewui Arcade "available" filter, if I removed the roms from their folder.

**Comment by dankan1890** [comment]
You have performed an audit (F1 or Shift-F1) of the test build?

With each change of version (also "r" for to the Test build), MEWUI not reload the previous available (to avoid conflicts), it only scan the rompaths.

Once performed the audit, at starting, MEWUI reload the availables directly from the saved file, without effective control if they exist or not (the same behavior that the MameUI with F5).

**Comment by neosquid** [comment]
Yes, I noticed the F1 audit option. It seems to have no effect, the number of games in my list stays the same after the audit completes.

**Comment by dankan1890** [comment]
Try to replace the test version with the official 1.66, it will take a while to start.
If the issue persists, check the romspath in the mame.ini.

**Comment by neosquid** [comment]
Yes, the official release version works fine if I drop the 1.66  mewui.exe into the mewui test folder. 

The mame,ini also looks fine, pointing to the correct rom directory.

**Comment by dankan1890** [comment]
Hi neosquid,
I tried downloading the latest version, unzipped it in a folder on the desktop, it doesn't reply the exposed issue.
For the moment, I close the issue as unreproducible. If the problem occurs again with the stable version, reopen it :)

**Comment by neosquid** [comment]
Still have the issue: did the following narrower test with less rom zips to compare the release and oct 19th test build

Windows 10 64 Bit

Tested: 
1. mewui test version (64bit) unzipped to desktop
2. mewui 0.166 (32bit) release unzipped to desktop

I added 4 roms to each roms folder within the above mewui version folders on my desktop

roms: xevious.zip , xexes.zip, xmcota.zip , xmen.zip
1. mewui 0.166 release detects all 4 roms plus breakout, pong,breakout
2. mewui test version detects only xmcota.zip
